marlowe-dashboard-client/tailwind.config.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
       purple: "#4700c3",
       lightpurple: "#8701fc",
       grayblue: "#f5f9fc",
-      purple: "#4c41e5",
+      purple: "#4700C3",
```

Comment:
Oops, we've got two purples here, don't know how that happened. But I think you've deleted the wrong one - in Zeplin, it's `#4c41e5`.

Reply:
sorry, my bad - it's `#4700c3` in the newer branded Zeplin file